### PR TITLE
Expose Target field in ScrollContainer

### PR DESCRIPTION
**Pull request by EVAST9919**

### CODE REVIEW

**Comment by iiSaLMaN** [comment]
`Target` field is the target value for `Current` to approach to, which sounds weird to be exposed publicly. Can you please provide the reason for exposing such a field?

**Comment by EVAST9919** [comment]
It will help me with https://github.com/ppy/osu/pull/7972. I need to know at which point the target will be changed.

**Comment by iiSaLMaN** [comment]
Assuming this is about hiding the scroll-to-top button if it's already pressed, you probably want to add a `IsScrolledToStart` method instead.

**Comment by EVAST9919** [comment]
No. My example: 
Screenshot_1

File: osu.Framework/Graphics/Containers/ScrollContainer.cs [hunk 1]
```diff
@@ -118,7 +118,7 @@ public bool ScrollbarOverlapsContent
         /// <summary>
         /// The target scroll position which is exponentially approached by current via a rate of distanceDecay.
         /// </summary>
-        private float target;
+        public float Target { get; private set; }
```
Comment (iiSaLMaN) [inline osu.Framework/Graphics/Containers/ScrollContainer.cs]:
```suggestion
        protected float Target { get; private set; }
```

Comment (EVAST9919) [reply osu.Framework/Graphics/Containers/ScrollContainer.cs]: Sounds fair

**Pull request merged**